tests/StoringGroceries.tex

```diff
@@ -7,18 +7,22 @@ \subsection{Goal}
 \subsection{Focus}
 This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
 
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \begin{minipage}{0.70\textwidth}
 	\subsection{Setup}
 	\begin{enumerate}
-		\item \textbf{Location:} One of the bookcases or cupboards in the apartment is used for this test, one where a table is near or can be put. 
-		\item \textbf{Start position:} The robot will start between the cupboard and the table in a random orientation, but facing towards the Cupboard.
-		\item \textbf{Cupboard:} The cupboard has 5 shelves between 0.30m and 1.80m from the ground and contains several objects (See \ref{rule:scenario_objects}).
+		\item \textbf{Location:} This test can take place either inside or outside the arena. The testing area must have a bookcase or cupboard, and a newarby table. The maximum distance between the Table and the Cupboard is 2 meters.
+		\item \textbf{Start position:} The robot starts between the cupboard and the table in a random orientation, but facing towards the Cupboard.
+		\item \textbf{Cupboard:} The cupboard has 5 shelves between 0.30m and 1.80m from the ground and contains several objects grouped by category or likeliness (See \ref{rule:scenario_objects}). The cupboard has at least one free space for starting a new set.
 		\begin{itemize}
 		 	\item \textbf{Door:} The cupboard has a single door, which is closed initially.
 		 	This door encloses some of the objects, covering up to one half of the cupboard (e.g. the left or bottom half), as indicated by the hatched area in Figure \ref{fig:storing_groceries_shelf}.
-		\end{itemize} 
-		\item \textbf{Table:} A table near to the Cupboard has 10 objects (See \ref{rule:scenario_objects}). If not all objects fit on the table, they will be added during the test. The maximum distance between the Table and the Cupboard is 2m.
-		\item \textbf{Objects:} Objects in the Cupboard and on the Table can be known, alike, or unknown. Also, there will be more than one object on each shelf. 
+		\end{itemize}
+		\item \textbf{Table:} The table may have up to 10 objects. If not all objects fit on the table, they will be added as the robot frees up space.
```

Comment:
May change to at least 5, cause only 5 objects are awarded for storing?

Reply:
There is a commented line over there for 6+ objects in the scoresheet. But let's be realistic, no robot can more more than 1 object per minute.

Why not _at least 5_ because it might be that the table can't hold all 5 at once. Indeed we can reduce the number of objects to max 5 **but** you want to be sure that robots can recognize as many objects as possible in real time.

Reply:
10 is more cluttered (interesting) but robots could select, out of the 10, the items they can manipulate etc. 
But indeed 'at least 5' would be OK

Reply:
Done!